src/projects/project-activity.html

```diff
@@ -30,8 +30,6 @@
 <link rel="import" href="../../bower_components/iron-icons/iron-icons.html">
 
 <link rel="import" href="../ajax/github-authenticated-ajax.html">
-<link rel="import" href="../list-items/list-item-label.html">
-<link rel="import" href="../list-items/column-list-item.html">
```

Comment:
Yes, they both are not used in that document.
Shouldn't have committed it with these changes though.